src/scheduler.jl

```diff
@@ -82,6 +112,10 @@ function compute_dag(ctx, d::Thunk; options=SchedulerOptions())
 
     node_order = x -> -get(ord, x, 0)
     state = start_state(deps, node_order)
+    # setup worker-to-scheduler channels
+    for p in ps
+        state.worker_chans[p.pid] = fetch(@spawnat p.pid RemoteChannel(1))
```

Comment:
```suggestion
        state.worker_chans[p.pid] = remotecall_fetch(RemoteChannel, p.pid,1)
```

Reply:
Addressed in latest push, although the suggestion doesn't work except on recent Julia versions, because `remotecall_fetch` didn't previously accept constructors as the first argument.